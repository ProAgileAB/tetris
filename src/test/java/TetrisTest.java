import org.approvaltests.Approvals;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class TetrisTest {

    @BeforeEach
    void setUp() {
    }

    @AfterEach
    void tearDown() {
    }

    @Test
    void upPressed() {
        assertTrue(true);
    }

    @Test
    void approvalTest() {
        Approvals.verify("hello approvals");
    }
}