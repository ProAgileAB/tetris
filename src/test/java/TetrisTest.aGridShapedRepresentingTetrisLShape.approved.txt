Grid of 3 rows, 2 cols:
#.
#.
##
